src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure implements Drawable, AreaCalculable {
+    private double side;
+
+    public Square(double side, String color) {
+        super.setColor(color);
+        this.side = side;
+    }
+
+    @Override
+    public double area() {
+        return Math.pow(side,2);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: square, area:" + area() + "sq. units, side: "
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create a variable for the area calculation and pass it to the System.out.println for better readability.